crosstool/cc_toolchain_config.bzl

```diff
@@ -44,6 +45,48 @@ _DYNAMIC_LINK_ACTIONS = [
     _OBJCPP_EXECUTABLE_ACTION_NAME,
 ]
 
+def _sdk_framework_dir(platform_type, sdk_version):
+    if platform_type == apple_common.platform_type.ios and sdk_version.compare_to(apple_common.dotted_version("9.0")) < 0:
```

Comment:
We shouldn't need this any more?

Reply:
ah yea, removed, note this is copied from bazel